server/README.md

```
@@ -117,81 +117,165 @@ The client to server commands and their format is as below
+
+#### Post Authentication Messages
+
+| Commands to server| Description|
+|--------------|------------------|
+| ChangePassword **old_password** **new_password** | Change password while logged in
+| Seek **size** **time** **incr** **W/B** | V1 Seeks a game of board size **size** with time per player **time** specified in seconds, increment per move **incr** specified in seconds and an optional choice of color **W** for white, **B** for black |
+| Seek **size** **time** **incr** **W/B/A** **komi** **pieces** **capstones** **unrated** **tournament** **opponent**| V2 Seeks a game, **A** is now used for random color, adding options for **komi** and custom piece counts, **unrated** and **tournament** are flags (either *1* or *0*), **opponent** is the name of the opponent allowed to join, blank to allow anyone to join  |
+| Seek **size** **time** **incr** **W/B/A** **komi** **pieces** **capstones** **unrated** **tournament** **extra_time_trigger** **extra_time_amount** **opponent** | V3 Seeks a game, **extra_time_amount** is extra time to add when move trigger is hit, **extra_time_trigger** is the move number on when to add the extra time to clock|
+| Seek **size** **time** **incr** **scale_increment** **W/B/A** **komi** **pieces** **capstones** **unrated** **tournament** **extra_time_trigger** **extra_time_amount** **opponent** | V4 Seeks a game, **scale_increment** is a flag (either *1* or *0*), if set, the increment is scaled by the move count (1-indexed) |
+|Rematch **game_id** **size** **time** **increment** **new_color** **komi** **pieces** **capstones** **unrated** **tournament** **extra_time_trigger** **extra_time_amount** **opponent** |V1 The client can send the rematch command using the last game id and previous game parameters, which creates a new seek with a rematch ID using the passed in game ID, so if both players send a rematch the server will send an "Accept Rematch **seek_id**" to the player that last sent the rematch command to accept the seek using the **seek_id**
+|Rematch **game_id** **size** **time** **increment** **scale_increment** **new_color** **komi** **pieces** **capstones** **unrated** **tournament** **extra_time_trigger** **extra_time_amount** **opponent** |V2 of Rematch command, adds **scale_increment**
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)